datafusion/proto/src/from_proto.rs

```diff
@@ -806,11 +806,20 @@ pub fn parse_expr(
                         .ok_or_else(|| Error::unknown("BuiltInWindowFunction", *i))?
                         .into();
 
+                    let args = match parse_optional_expr(&expr.expr, registry)? {
+                        None => {
+                            vec![]
+                        }
+                        Some(x) => {
+                            vec![x]
+                        }
+                    };
+
                     Ok(Expr::WindowFunction {
                         fun: datafusion_expr::window_function::WindowFunction::BuiltInWindowFunction(
                             built_in_function,
                         ),
-                        args: vec![parse_required_expr(&expr.expr, registry, "expr")?],
```

Comment:
Unlike  WindowFunction  with `AggrFunction`  like `min(col)`. `max(col)`. `sum(col)` it must have required_exp

Built-in WindowFunction like `ROW_NUMBER()` . `RANK()`. `DENSE()`   have to have this value